OpenRA.Mods.Common/Traits/Render/RenderSprites.cs

```diff
@@ -34,10 +34,10 @@ public class RenderSpritesInfo : IRenderActorPreviewInfo, ITraitInfo
 		public readonly Dictionary<string, string> RaceImages = null;
 
 		[Desc("Custom palette name")]
-		public readonly string Palette = null;
+		[PaletteReference] public readonly string Palette = null;
 
 		[Desc("Custom PlayerColorPalette: BaseName")]
-		public readonly string PlayerPalette = "player";
+		[PaletteReference] public readonly string PlayerPalette = "player";
```

Comment:
... and so on for these other player palette references (no need to spam up the discussion listing all of them).